Initium-Odp/war/WEB-INF/odppages/viewitemmini.jsp

```diff
@@ -48,6 +48,7 @@
 				
 				<c:forEach items="${item.slots}" var="slot">
 				<c:if test="${slot.slotIsEmpty == true}"><div class='socket-base' minitip='${slot.slotName}<br>${slot.slotTooltip}'></div></c:if>
+				<c:if test="${slot.slotIsEmpty == false}"><div class='socket-base'></div><div class = 'socket-item' <img src='${slot.slotIcon}'/> minitip='${slot.slotName}<br>${slot.slotModifiers}'></div></c:if>

```

Comment:
You can't just output a list as you have done here with slot.modifiers. You have to go through them with a for loop and output each element one by one. I would put each string wrapped in it's own <div> element too to make sure each one is on it's own line.